modules/neo4j/src/test/java/org/testcontainers/containers/Neo4jContainerTest.java

```diff
@@ -260,11 +260,13 @@ public void shouldDefaultExportBoltHttpAndHttps() {
 
     @Test
     public void shouldRespectCustomWaitStrategy() {
-        Neo4jContainer<?> neo4jContainer = new Neo4jContainer<>("neo4j:4.4").waitingFor(new CustomDummyWaitStrategy());
-
-        neo4jContainer.configure();
-
-        assertThat(neo4jContainer.getWaitStrategy()).isInstanceOf(CustomDummyWaitStrategy.class);
+        try (
+            Neo4jContainer<?> neo4jContainer = new Neo4jContainer<>("neo4j:4.4")
+                .waitingFor(new CustomDummyWaitStrategy())
+        ) {
+            neo4jContainer.start();
```

Comment:
I think this test became a bit awkward like this. Why is this change even necessary as part of this PR?